.github/workflows/main.yml

```diff
@@ -59,6 +59,9 @@ jobs:
           - os: macOS
             feature: metal
             device: apple_m2_pro
+          - os: Linux
+            feature: cuda
+            device: graviton2
```

Comment:
Wouldn't the device be `nvidia_rtx_????` here?

Reply:
AFAIK, graviton represents the CPU, but we still have NVidia gear that we're running accelerated code on

Reply:
yeah you're right, I've updated the tag to be: `nvidia_tesla_t4g`